docs/cluster_spec.md

```diff
@@ -742,6 +742,23 @@ spec:
     logFormat: json
 ```
 
+### Graceful Node Shutdown
+
+Graceful node shutdown allows kubelet to prevent instance shutdown until Pods have been safely terminated or a timeout has been reached.
+
+By default, kOps will try to add a 30 second timeout for 30 seconds where the first 2o seconds is reserved for normal Pods and the last 10 seconds for critical Pods.
+
+This configuration can be changed as follows:
+
+```yaml
+spec:
+  kubelet:
+    shutdownGracePeriod: 60s
+    shutdownGracePeriodCriticalPods: 20s
```

Comment:
To match defaults:
```suggestion
    shutdownGracePeriod: 30s
    shutdownGracePeriodCriticalPods: 10s
```

Reply:
The point was to show how to change the defaults. so feel like this can/should stay as-is.